binding/web/README.md

```diff
@@ -91,21 +91,45 @@ npx pvbase64 -h
 
```

Comment:
in `### Public Directory` and `### Base64`, we should also say something about the context files in addition to the parameter model file

Reply:
Good point!